src/System.Memory/ref/System.Memory.cs

```diff
@@ -53,6 +53,9 @@ public static partial class MemoryExtensions
         public static int IndexOf<T>(this System.Span<T> span, System.ReadOnlySpan<T> value) where T : System.IEquatable<T> { throw null; }
         public static int IndexOf<T>(this System.Span<T> span, T value) where T : System.IEquatable<T> { throw null; }
         public static bool IsWhiteSpace(this System.ReadOnlySpan<char> span) { throw null; }
+#if !FEATURE_PORTABLE_SPAN
```

Comment:
It is weird to use `FEATURE_PORTABLE_SPAN` for this because of this API has nothing to do with portable span.